tests/test_lint.py

```diff
@@ -188,47 +189,44 @@ def test_sphinx_md_files(self):
     # SPECIFIC LINT TESTS #
     #######################
     from .lint.actions_awsfulltest import (
-        test_actions_awsfulltest_warn,
-        test_actions_awsfulltest_pass,
         test_actions_awsfulltest_fail,
+        test_actions_awsfulltest_pass,
+        test_actions_awsfulltest_warn,
     )
-    from .lint.actions_awstest import test_actions_awstest_pass, test_actions_awstest_fail
-    from .lint.files_exist import (
-        test_files_exist_missing_config,
-        test_files_exist_missing_main,
-        test_files_exist_depreciated_file,
-        test_files_exist_pass,
+    from .lint.actions_awstest import (
+        test_actions_awstest_fail,
+        test_actions_awstest_pass,
     )
     from .lint.actions_ci import (
-        test_actions_ci_pass,
-        test_actions_ci_fail_wrong_nf,
         test_actions_ci_fail_wrong_docker_ver,
+        test_actions_ci_fail_wrong_nf,
         test_actions_ci_fail_wrong_trigger,
+        test_actions_ci_pass,
     )
-
     from .lint.actions_schema_validation import (
         test_actions_schema_validation_missing_jobs,
         test_actions_schema_validation_missing_on,
         test_actions_schema_validation_fails_for_additional_property,
```

Comment:
And finally we can fix the issue.

```suggestion
        test_actions_schema_validation_fails_for_additional_property,
        test_actions_schema_validation_missing_jobs,
        test_actions_schema_validation_missing_on,
```

Reply:
[f82ea0a](https://github.com/nf-core/tools/pull/1538/commits/f82ea0ac130f8e05bae96ed03922171fe6a8d229) just solved that.